docs/release-management.md

```diff
@@ -63,6 +63,7 @@ Also, as soon as we start working on the next major version, we need to require
 * Travis CI, see what we did for Matomo 3 in https://github.com/matomo-org/matomo/pull/8452#discussion-diff-35731015L93 and https://github.com/matomo-org/travis-scripts/pull/53
 * Create the `PULL_REQUEST_TEMPLATE` similar to https://github.com/matomo-org/matomo/pull/12412/files
 * Once the build for 4.x-dev succeeds, make it the default branch for Matomo and all plugins
+* Update the [submodule github action](https://developer.matomo.org/guides/release-management#when-starting-to-work-on-the-next-major-release) to use the new main branch
```

Comment:
Actually there are more than the `submodules update` action that need to be adjusted. But guess we can also have a look if it's easy to change the actions so they are using the configured default branch...

Reply:
@sgiehl which other ones would we need to adjust? We could simply document them here quick and then it be easy to adjust them when needed

Reply:
[submodule action](https://github.com/matomo-org/matomo/blob/4.x-dev/.github/workflows/submodules.yml#L28)
[composer update action](https://github.com/matomo-org/matomo/blob/4.x-dev/.github/workflows/composer-update.yml#L28)
And the CLDR action as soon as it was merged.